.github/workflows/snapshot-build.yml

```diff
@@ -30,10 +30,10 @@ jobs:
           - arch: linux/amd64
             profile: production
             suffix: ubuntu-x86_64-${{ github.ref_name }}
-#          - arch: linux/arm64
-#            # Fewer optimizations on aarch64 to reduce CI time
-#            profile: release
-#            suffix: ubuntu-aarch64-${{ github.ref_name }}
+          - arch: linux/arm64
+            # Build at least something for aarch64 that doesn't exceed CI's time limit
+            profile: dev
```

Comment:
Ew, I get why we do that, but wouldn't it be better to disable it for now and just setup aarch64 runner?

Reply:
It would be nice to have aarch64 runner, but at least we'll have something in the meantime.
Alternatively I can build images and push them from my machine :man_shrugging:

Reply:
Yeah, that would make more sense. I think optimizations would be critical for RPI.

Reply:
I'm curious who will use this dev build? :eyes:

Reply:
Those who are on aarch64 platform. Many crucial packages are already compiled with `-O3` even in debug, so it isn't as bad as you might think naively.